filebeat/input/v2/input-cursor/input.go

```diff
@@ -25,12 +25,12 @@ import (
 
 	"github.com/urso/sderr"
 
-	"github.com/elastic/go-concert/ctxtool"
-	"github.com/elastic/go-concert/unison"
-
 	input "github.com/elastic/beats/v7/filebeat/input/v2"
 	"github.com/elastic/beats/v7/libbeat/beat"
+	"github.com/elastic/beats/v7/libbeat/common/acker"
 	"github.com/elastic/beats/v7/libbeat/logp"
+	"github.com/elastic/go-concert/ctxtool"
```

Comment:
These `github.com/elastic/go-concert/*` imports seem to be in the incorrect place.